cmd/liqo-controller-manager/main.go

```diff
@@ -131,6 +150,8 @@ func main() {
 		os.Exit(1)
 	}
 
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
```suggestion
	ctx = ctrl.SetupSignalHandler()
```

Then, it is simply given this context to the manager when started